lib/tasks/key_store/active_record.rake

```diff
@@ -10,6 +10,7 @@ namespace :skl do
 
       namespace :clear do
         task stale: :environment do
+          return unless SurrogateKeyLogging.config.enabled
```

Comment:
Minor, thoughts on writing some `puts` statement to make it clear why nothing is getting cleared when the rake task runs?

Reply:
done added.